modules/microshift-applying-manifests-example.adoc

```diff
@@ -29,7 +29,10 @@ $ sudo mkdir -p ${MANIFEST_DIR}
 [source,terminal]
 ----
 $ sudo tee ${MANIFEST_DIR}/busybox.yaml &>/dev/null <<EOF
-
+----
++
+[source,terminal]
+----
```

Comment:
```suggestion
```

Lines 31-62 are all part of 1 command and need to appear together.

ht @dhensel-rh